test.js

```diff
@@ -11,5 +11,11 @@ test('main', t => {
 	t.deepEqual(Object.keys(importModules('fixture', {camelize: false, fileExtensions: ['.xjs']})), ['non-standard']);
 	t.deepEqual(Object.keys(importModules()), ['index', 'package']);
 	t.deepEqual(importModules('fixture/empty'), {});
-	t.deepEqual(importModules('non-existent'), {});
+});
+
+test('non-existent directory', t => {
+	const error = t.throws(() => {
```

Comment:
Prefer using the `t.throws` native error message matching (you can use a regex)

Reply:
I looked into regex, but `String` has `includes` built in so why not use it? Can `includes` be used in `t.throws` as a matcher?

Reply:
Nevermind, I just got the regex working. Pushing now.